style.css

```diff
@@ -0,0 +1,50 @@
+body{
+    padding:0;
+    margin:0;
+    font-family:"Arial",sans-serif;
+}
+
+
+.container{
+    width:100%;
+    max-width:1024px;
+    margin-left:auto;
+    margin-right:auto;
+    padding:10px;
+    background-color: rgba(233, 249, 45, 0.22);
+}
+
+a{
+    color: rgba(23, 174, 0, 0.78);
+    text-decoration:none;
+}
+
+a:hover{
+    box-shadow:0 0px 5px 0 black;
+}
+
+.photo{
+    float:right;
+    width:150px;
+    height:200px;
+    margin:5px;
+    overflow:hidden;
+    background: url("./assets/portrait.jpg") 0 0;
+    background-size: 105%;
+}
+
+.item-name{
+    width:180px;
+    height:30px;
+    font-weight:bold;
+}
+
+.item h3{
+    margin-bottom:0;
+    padding-bottom:5px;
+}
+
+.item>p{
```

Comment:
Before and after the combinator between selectors (for example, p > a) one space is put. See more in this guide: http://codeguide.academy/html-css.html#html-language